src/player.js

```diff
@@ -1183,24 +1184,20 @@ export default class Player extends FakeEventTarget {
    * @returns {void}
    */
   notifyExitFullscreen(): void {
-    if (this._fullscreen) {
-      this._fullscreen = false;
+    if (!this.isFullscreen()) {
       this.dispatchEvent(new FakeEvent(CustomEventType.EXIT_FULLSCREEN));
     }
   }
 
   /**
    * Request the player to enter fullscreen.
    * @public
+   * @param {HTMLElement} fullScreenElement - set html element to full screen
    * @returns {void}
    */
-  enterFullscreen(): void {
-    if (!this._fullscreen) {
-      if (this.isInPictureInPicture()) {
-        this.exitPictureInPicture();
-      }
-      this.dispatchEvent(new FakeEvent(CustomEventType.REQUESTED_ENTER_FULLSCREEN));
-    }
+  enterFullscreen(fullScreenElement: ?HTMLElement): void {
+    this.dispatchEvent(new FakeEvent(CustomEventType.REQUESTED_ENTER_FULLSCREEN));
```

Comment:
still needed?

Reply:
It's not needed but I keeped it for external listeners